common/src/main/java/gov/cms/ab2d/common/service/JobServiceImpl.java

```diff
@@ -90,13 +94,15 @@ public Resource getResourceForJob(String jobUuid, String fileName) throws Malfor
         }
 
```

Comment:
Add job's UUID to MDC as well

Reply:
Ok, same with this one, should be called by downloadFile